package.json

```diff
@@ -35,40 +35,40 @@
     "yargs": "13.2.4"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/plugin-proposal-class-properties": "^7.5.5",
-    "@babel/plugin-proposal-export-default-from": "^7.5.2",
-    "@babel/plugin-proposal-export-namespace-from": "^7.5.2",
+    "@babel/core": "^7.7.5",
+    "@babel/plugin-proposal-class-properties": "^7.7.4",
+    "@babel/plugin-proposal-export-default-from": "^7.7.4",
+    "@babel/plugin-proposal-export-namespace-from": "^7.7.4",
     "@babel/plugin-syntax-export-extensions": "^7.0.0-beta.32",
-    "@babel/plugin-transform-react-constant-elements": "^7.5.0",
-    "@babel/plugin-transform-react-inline-elements": "^7.2.0",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/register": "^7.5.5",
-    "@easypost/build": "^2.2.2",
-    "babel-eslint": "10.0.1",
-    "babel-loader": "8.0.6",
-    "babel-plugin-istanbul": "5.1.4",
+    "@babel/plugin-transform-react-constant-elements": "^7.7.4",
+    "@babel/plugin-transform-react-inline-elements": "^7.7.4",
+    "@babel/preset-env": "^7.7.6",
+    "@babel/register": "^7.7.4",
+    "@easypost/build": "^2.4.2",
+    "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.0.6",
+    "babel-plugin-istanbul": "^5.1.4",
     "babel-plugin-transform-export-extensions": "^6.22.0",
-    "babel-preset-minify": "0.5.0",
-    "babel-register": "6.24.1",
-    "chai": "4.2.0",
-    "chai-as-promised": "7.1.1",
-    "cross-env": "5.2.0",
+    "babel-preset-minify": "^0.5.0",
+    "babel-register": "^6.24.1",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "cross-env": "^5.2.0",
     "eslint": "^4.19.1",
     "eslint-config-airbnb": "^16.1.0",
-    "eslint-import-resolver-webpack": "0.11.1",
-    "eslint-plugin-import": "2.17.3",
-    "eslint-plugin-jest": "^22.15.2",
+    "eslint-import-resolver-webpack": "^0.11.1",
+    "eslint-plugin-import": "^2.17.3",
+    "eslint-plugin-jest": "^22.21.0",
     "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.14.3",
-    "mocha": "6.1.4",
+    "eslint-plugin-react": "^7.17.0",
+    "mocha": "^6.1.4",
     "npm-audit-ci": "^1.2.1",
-    "nyc": "14.1.1",
-    "sinon": "7.3.2",
-    "sinon-chai": "3.3.0",
-    "vows": "0.8.2",
+    "nyc": "^14.1.1",
+    "sinon": "^7.3.2",
+    "sinon-chai": "^3.3.0",
+    "vows": "^0.8.2",
     "webpack": "^4.39.3",
-    "webpack-cli": "^3.3.7",
-    "webpack-node-externals": "1.7.2"
+    "webpack-cli": "^3.3.10",
+    "webpack-node-externals": "^1.7.2"
```

Comment:
curious, why were only some deps pinned?

Reply:
We used to pin everything, and a few things probably slipped through. There's no need for pinning in libraries, though, so I've removed all of the pins.